dts/bindings/mtd/tenstorrent,fixed-partitions.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2025 Tenstorrent AI ULC
```

Comment:
I think we can likely drop this binding, since "padto" (i.e. partition size) is already captured by the upstream bindings.

It might be possible to use a partition-label-based convention for the executable bit as well, which we should be able to handle in python.